### DISCUSSION TITLE
train.py 아님 모듈화?

### DISCUSSION THREAD

**Discussion by rachel318318** [potato-farm/alchera-ai-challenge#4] [Q&A] [answered]
training 파이프라인 모듈화 말고 py파일 하나만 내는가?

**Comment by 8639sung** [comment]
답변 온 건가요??

**Reply by rachel318318** [reply]
아 아직 질문도 안올렸는데(삐질
지금 바로 올리겠습니다 ㅋㅋ

**Reply by 8639sung** [reply]
넵!!! 감사합니다. 연락 오면 알려주세요!! 문의말고 게시글로 올리는게 답변 더 빠르게 받을 수 있을 것 같아요!

**Comment by rachel318318** [comment, marked as answer]
답 왔습니다!

> 안녕하세요 하리보님, 문의 남겨주셔서 감사합니다.
모듈화 하셔도 상관 없습니다. 안내드린 파일 목록은 꼭 포함하여야 하는 파일을 말씀 드린 것이고 모듈화 하셔도 상관 없습니다.
train 파이프라인은 추후 학습 방법에 대해서 확인하는 용도로만 사용할 예정이며, 평가는 test.py 파일로만 이루어집니다.
그럼 멋진 결과물 기대하겠습니다. 좋은 하루 되세요.
